hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2966,6 +2966,19 @@ public String getRpcPort() {
   }
 
   @Override
+
+  public String getOmRatisRoles() {
+    List<ServiceInfo> serviceList = null;
+    int port = omNodeDetails.getRatisPort();
+    try {
+      serviceList = getServiceList();
+    } catch (IOException e) {
+      LOG.error("IO-Exception Occurred");
```

Comment:
better to log the exception
```suggestion
      LOG.error("IO-Exception Occurred", e);
```

Reply:
Will do, thanks for the suggestion !!